browser/typescript/v2/react-app/src/ampli/index.ts

```diff
@@ -0,0 +1,747 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Ampli - A strong typed wrapper for your Analytics
+ *
+ * This file is generated by Amplitude.
+ * To update run 'ampli pull browser-ts-ampli-v2'
+ *
+ * Required dependencies: @amplitude/analytics-browser@^0.5.1
+ * Tracking Plan Version: 0
+ * Build: 1.0.0
+ * Runtime: browser:typescript-ampli-v2
+ *
+ * [View Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/latest)
+ *
+ * [Full Setup Instructions](https://data.amplitude.com/test-codegen/Test%20Codegen/implementation/browser-ts-ampli-v2)
+ */
+
+import * as amplitude from '@amplitude/analytics-browser';
+
+export type AmplitudeClient = typeof amplitude;
+
+export type Environment = 'development' | 'production';
+
+export const ApiKey: Record<Environment, string> = {
+  development: '',
+  production: '',
+};
+
+/**
+ * Default Amplitude configuration options. Contains tracking plan information.
+ */
+export const DefaultOptions: BrowserOptions = {
+  plan: {
+    version: '0',
+    branch: 'main',
+    source: 'browser-ts-ampli-v2',
+    versionId: '79154a50-f057-4db5-9755-775e4e9f05e6',
+  }
+};
+
+export interface LoadOptions {
+  environment?: Environment;
+  disabled?: boolean;
+  client?: {
+    apiKey?: string;
+    options?: BrowserOptions;
```

Comment:
Can we rename the key to configuration in order to align with other ampli sdk?

Reply:
Which keys do we need to rename?

Reply:
`options?: BrowserOptions;` to `configuration?: BrowserOptions`